aten/src/ATen/native/native_functions.yaml

```diff
@@ -3868,6 +3868,7 @@
   dispatch:
```

Comment:
should we fix other variants of min/max above?

Reply:
this is calling legacy code, and I'm eventually calling that code as well.
do you mean move these out of legacy?

Reply:
what I mean is - do we want to add qtensor support for these operations too?

Reply:
oh, the others are elementwise max/min, we probably don't need them right now, we can add them later if there is a need.